subprojects/docs/src/docs/release/notes.md

```diff
@@ -105,6 +105,14 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.html) for all supported methods.
 
 
+<a name="build-init"></a>
+### Build Init improvements
+
+#### Build Init utilizing version Catalogs
+The [build init plugin](userguide/build_init_plugin.html) now generates projects utilizing version catalogs using the conventional location `gradle/libs.versions.toml`. 
+Check the relevant part of the [user manual](userguide/platforms.html#sub:central-declaration-of-dependencies) and the [version catalog TOML file format](userguide/platforms.html#sub::toml-dependencies-format) for more information on the topic.
```

Comment:
```suggestion
This was done to encourage version catalog usage since it became a stable feature in Gradle 8.0.

Refer to the [user manual](userguide/platforms.html#sub:central-declaration-of-dependencies) and the [TOML file format](userguide/platforms.html#sub::toml-dependencies-format) for information on the topic.
```